lib/collections.js

```diff
@@ -27,12 +27,23 @@ var normalizedWords = [];
 // The words stored as keys of a dictionary
 var uniqueWordsAsKeys = {};
 
+for (var i = 0; i < normalizedWords.length; i++) {
+  var specialWords = normalizedWords[i];
+  uniqueWordsAsKeys[specialWords] = true;
+}
+
+console.log(uniqueWordsAsKeys);
+
 // The keys pulled out into a list
 var uniqueWords = [];
 
-/* your code starts here */
+var j = 0;
 
-/* your code ends here */
+for (var key in uniqueWordsAsKeys) {
+  uniqueWords[j++] = key;
```

Comment:
Aaah. Aaah. 
Don't do this - this is 'clever' code. Just use `.push()` to add things to the array.